test_vm/tests/batch_onboarding.rs

```diff
@@ -50,11 +51,30 @@ impl Onboarding {
 fn batch_onboarding(v2: bool) {
     let store = MemoryBlockstore::new();
     let v = TestVM::<MemoryBlockstore>::new_with_singletons(&store);
-    let addrs = create_accounts(&v, 1, &TokenAmount::from_whole(10_000));
     let seal_proof = RegisteredSealProof::StackedDRG32GiBV1P1;
+
+    let proven_count = batch_onboarding_test(&v, v2, &seal_proof);
+
+    let network_stats = v.get_network_stats();
```

Comment:
`get_network_stats` should be implementable outside the VM, rather than as a method, in which case you can push this down into the test method.